app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/DeterminingIfOrganisationIsTrustController.scala

```diff
@@ -193,6 +195,7 @@ class DeterminingIfOrganisationIsTrustController @Inject()(
     }
   }
 
+  @SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
```

Comment:
do you need this suppress warnings?

Reply:
not anymore as the audit event call has been removed